src/services/DocumentSharing/index.tsx

```diff
@@ -6,11 +6,22 @@ export interface StorageApiResponse {
   key: string;
 }
 
-export const uploadDocument = async (document: Document): Promise<string> => {
+const DEFAULT_TTL_MS = 20 * 1000;
```

Comment:
20 seconds seemed too short for the offline feature to be any useful. Tried it out live and the app and it seemed like I’m rushing to go back to see a QR still there when I’m offline